src/components/commentary/Commentary.js

```diff
@@ -240,7 +240,7 @@ const Commentary = (props) => {
       if (commentaryObject.bookIntro) {
         commText += "<p>" + changeBaseUrl(commentaryObject.bookIntro) + "</p>";
       }
-      if (commentaryObject?.commentaries?.length > 0) {
+       if (commentaryObject?.commentaries?.length > 0 || commentaryObject?.bookIntro) {
```

Comment:
HI Beena for John 1 it doesnt show the message as the bookIntro is a array can you change the check to the code as below it will handle both string and array cases
```suggestion
      if (commentaryObject?.commentaries?.length > 0 || commentaryObject?.bookIntro?.length > 0) {
```